magento/magento2
# Remove magento info in production mode

### Description (*)

Remove magento version and edition when access controller magento_version in production mode 

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Set magento use production mode
2. Go to https://magento2.test/magento_version
3. Should not display magento version and editor 

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39506: Remove magento info in production mode
